module/preference/PreferenceDialog.kt

```diff
@@ -472,22 +505,18 @@ object PreferenceDialog {
                         id = PreferenceDialog.javaClass.canonicalName + ".secondary",
                         initSize = Either.first(PREFERENCE_GROUP_INIT_SIZE), minSize = PREFERENCE_GROUP_MIN_SIZE
                     ) {
-                        Column(modifier = Modifier.fillMaxHeight().padding(10.dp)) {
-                            if (focusedPreferenceGroup.name.isBlank()) {
-                                focusedPreferenceGroup = state.rootPreferenceGroup.entries.first()
-                            }
-                            focusedPreferenceGroup.Display()
+                        Column(modifier = Modifier.fillMaxHeight().padding(Dialog.DIALOG_SPACING)) {
+                            selectedPreferenceGroup.Display()
                         }
                     }
                 )
                 Separator.Horizontal()
-                ColumnSpacer()
+                ColumnSpacer(multiplier = 2)
                 Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.End) {
                     ChangeFormButtons(state)
-                    RowSpacer()
-                    RowSpacer()
+                    RowSpacer(multiplier = 2)
                 }
-                ColumnSpacer()
+                ColumnSpacer(multiplier = 2)
```

Comment:
We use these Column and Row spacers multiple times occassionally, so I've added a `multiplier` argument allowing us to declare 2 without actually having to state so twice.